.travis.yml

```
@@ -45,3 +45,23 @@ matrix:
   - name: "Clang Release"
     compiler: clang
     config: Release
+  - &freebsd_common
+    name: "Clang 10 + system libraries"
+    os: freebsd
+    compiler: clang
+    before_install:
+      - | # FreeBSD 12.2 has Clang 10 but TravisCI uses FreeBSD 12.1
+        if [[ "$($CC --version)" =~ 8.0 ]]; then
```

```suggestion
        if [[ $(${CC:-clang} --version) =~ 8.0 ]]; then
```

bash `[[` does not require you to quote the lhs for a lot of comparisons, also it would be a good idea to add an alternative replacement in the case when someone is copy-pasting commands to test or reproduce

---

Thanks. I mainly write in Ash (/bin/sh on FreeBSD), so rarely use `[[`.